homework6/Hangman/__main__.py

```diff
@@ -0,0 +1,243 @@
+import json
+import os
+import random
+
+from Hangman.hangman_image import hangman_image
+
+# Закидываем в словарь слова
+with open('Hangman/words.json', 'r', encoding='utf-8') as fh:
+    dictionary = json.load(fh)['words']
+
+
+class Game(object):
```

Comment:
https://ru.wikipedia.org/wiki/%D0%91%D0%BE%D0%B6%D0%B5%D1%81%D1%82%D0%B2%D0%B5%D0%BD%D0%BD%D1%8B%D0%B9_%D0%BE%D0%B1%D1%8A%D0%B5%D0%BA%D1%82